youtube_dl/extractor/sverigesradio.py

```diff
@@ -0,0 +1,107 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..compat import compat_str
+from ..utils import (
+    int_or_none,
+    try_get,
+)
+
+
+class SverigesRadioBaseIE(InfoExtractor):
+    _BASE_URL = 'https://sverigesradio.se/sida/playerajax'
+
+    def _extract_format(self, query, audio_id, audio_type, fatal=True):
+        audiourl = self._download_json(
+            self._BASE_URL + '/getaudiourl', audio_id,
+            fatal=fatal, query=dict(query, type=audio_type, quality='high', format='iis'))
+        if audiourl and ('audioUrl' in audiourl) and audiourl['audioUrl'] != "":
+            return [{'vcodec': 'none', 'url': compat_str(audiourl['audioUrl'])}]
```

Comment:
is there a need to use `compat_str` here?

Reply:
Not sure i follow the pattern when it's used in the other extractors. Looks like to potentially convert integer etc to string?